### PR TITLE
[feat] User profile & Updated Summary pages

### PR DESCRIPTION
**Pull request by Hamdrive**
### **Note to reviewer**: 

> 1. Kindly review files that are specific to this PR only, they begin with the word `UserProfile` or `ProfileNavbar` as rest of the files are minute changes/fixes/styles or updates from merging another PR to this branch.
> 2. Changes have also been made to `Summary` file, this is purely to reflect the addition of the ordered products
> 3. AddressBook has not been added to this PR, that is a feature for a future release and can be ignored from both code and deployment preview functionality

In this PR, I have successfully created a user profile page, which allows the user to directly access profile information/settings that matter.
Features:
1. 3 sections: Settings, Order History & Address Book
2. User logout and general information can be handled from Settings
3. ProtectedRoute, user can only view this page on successfull authentication
4. user can also view their order history from this page

## Deployment Preview
https://deploy-preview-29--fueledup-store.netlify.app/

### CODE REVIEW

**Review by prachi-sahani — COMMENTED** [review]
All the changes look good. Code is well structured and can be merged

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ export function Navbar() {
             </div>
           </div>
           <ul className={` ${"nav-menu"} h-100`}>
-            <li className="nav-item">
+            {/* <li className="nav-item">
```
Comment (shivampandey0) [inline src/components/Navbar.jsx]: Remove unused code comments.

File: src/pages/userProfile/UserProfileOrders.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { useProduct } from "../../context/product-context";
+import styles from "./UserProfile.module.css";
+
+export function UserProfileOrders({}) {
+  const {
+    state: { orders },
+  } = useProduct();
+
+  console.log(orders);
```
Comment (shivampandey0) [inline src/pages/userProfile/UserProfileOrders.jsx]: Remove `console.log()` from deployed code.

**Review by shivampandey0 — APPROVED** [review]
Clean code written, Liked the use of null check.

Good to merge

File: src/components/Navbar.jsx (hunk 1 above)
Comment (Hamdrive) [reply src/components/Navbar.jsx]: Cleaned up!

File: src/pages/userProfile/UserProfileOrders.jsx (hunk 2 above)
Comment (Hamdrive) [reply src/pages/userProfile/UserProfileOrders.jsx]: done!

**Comment by Hamdrive** [comment]
Successfully reviewed by 2 team members, changes suggested have been implemented and deployment is functional.
Thus closing PR and merging to dev

**Pull request merged**